TAF/BDT/Steps/JobListingTestsSteps.cs

```diff
@@ -0,0 +1,56 @@
+﻿using NUnit.Framework;
+using TAF.Core.Browser;
+using TAF.Core.Utilities;
+using TAF.Web.Pages;
+
+namespace TAF.Tests.BDT.Steps
+{
+    [Binding]
+
+    public class JobListingTestsSteps
+    {
+        private MainPage _mainPage;
+        private JobListingsPage _jobListingsPage;
+        private string defaultSearchMessage;
+
+        [Given(@"setup page objects for job listing tests")]
+        public void GivenSetupPageObjectsForJobListingTests()
+        {
+            _mainPage = new();
+            _jobListingsPage = new();
+        }
+
+        [Given(@"I navigate to Careers button")]
+        public void GivenINavigateToCareersButton()
+        {
+            Browser.NewBrowser.Action.MoveToElement(_mainPage.Header.CareersButton.OriginalWebElement).Build().Perform();
```

Comment:
It is better to create a method inside main page class that will move to element. Ex: _mainPage.Header.MoveToCareersButton()

Reply:
Or create ActionsHelper class. There should be as less Browser instances in steps as possible